test/jasmine/tests/plot_interact_test.js

```diff
@@ -58,8 +58,8 @@ describe('Test plot structure', function() {
                 Plotly.newPlot(gd, mockData, mockLayout).then(done);
             });
 
-            it('has one *subplot xy* node', function() {
-                expect(countSubplots()).toEqual(1);
+            it('has two *subplot xy* nodes', function() {
+                expect(countSubplots()).toEqual(2);
```

Comment:
How about modifying `countSubplots` function to return half of the layers or only select the background layers.
We could also have `countSubplotsBackgroung()` and `countSubplotsForeground()` instead.

Reply:
I opted for splitting it up into  `countSubplotsBackground`  and `countSubplotsForeground`